tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc

```diff
@@ -27,6 +27,25 @@
 #include "tensorflow/core/public/version.h"
 #include "tensorflow/core/util/env_var.h"
 
+#if TF_MAJOR_VERSION == 2 && TF_MINOR_VERSION == 4
+#define TF_2_4
+#endif
+
+#if defined(TF_1_12) || defined(TF_1_14) || defined(TF_2_4)
```

Comment:
How about using TF <= 2.8 ? otherwise building for other tf version will fail?

Reply:
done